translate/set.go

```diff
@@ -160,6 +160,24 @@ OUTER:
 	return ret
 }
 
+// Map returns a new TranslationSet with To translation paths further
+// translated through mappings.  Translations not listed in mappings are
+// copied unmodified.
+func (ts TranslationSet) Map(mappings TranslationSet) TranslationSet {
+	if mappings.FromTag != ts.ToTag || mappings.ToTag != ts.ToTag {
```

Comment:
When would we expect `mappings.ToTag == ts.ToTag` to be true? We use `mapping.From.String()` as keys for `ret.Set`, so wouldn't `mappings.FromTag == ts.ToTag` always need to be true? Thanks for any corrections

Reply:
Yes, both should be true.  Concretely, `ts` will typically map `yaml` -> `json`, and `Map` performs mappings in output space, so `mappings` should be `json` -> `json`.